### PR TITLE
fix: eth send very small amounts

### PR DESCRIPTION
**Pull request by martinkaintas**

### CODE REVIEW

File: src/protocols/ethereum/libs/EthereumAdapter.ts [hunk 1]
```diff
@@ -242,7 +242,7 @@ export class EthereumAdapter extends BaseProtocolAdapter {
     const web3Eth = new Web3Eth(nodeUrl);
     const nonce = await this.getTransactionCount(options.fromAccount.address!);
 
-    const hexAmount = bigIntToHex(BigInt(toWei(amount, 'ether')));
+    const hexAmount = bigIntToHex(BigInt(toWei(amount.toFixed(ETH_COIN_PRECISION), 'ether')));
```
Comment (CedrikNikita) [inline src/protocols/ethereum/libs/EthereumAdapter.ts:245]: Is `toWei` unable to handle E notation `1e-18`?

Comment (martinkaintas) [reply src/protocols/ethereum/libs/EthereumAdapter.ts:245]: Yes, it throws an error that it cannot convert it

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable.

**Comment by Liubov-crypto** [comment]
I can send 1e-20 ETH from my wallet. Transaction was successful: https://sepolia.etherscan.io/tx/0x219bcfe5bcc7f5e88b160152977fd031bc1892ef0f4d51edee4e6d497d47d367 
The wallet is extremely slow for me and I can see many errors in console: 
errrr

Also when I'm opening Send ETH modal I'm getting this error:

**Pull request merged**